### PR TITLE
Machines will drop their input and output slots when broken.

### PR DESCRIPTION
**Pull request by Riley31415**
Title

### CODE REVIEW

**Comment by Riley31415** [comment]
Also you should add hacktober tag to ur repo ;)

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>me.sfiguz7</groupId>
   <artifactId>ExtraTools</artifactId>
-  <version>1.2</version>
+  <version>MODIFIED</version>
```
Comment (Alessio-Colombo) [inline pom.xml]: \>:(

**Review by Alessio-Colombo — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (Riley31415) [reply pom.xml]: I mean... You use busy build page so might aswell >:)

**Comment by Riley31415** [comment]
Fine i fixed it :(

**Review by Alessio-Colombo — APPROVED** [review]

**Pull request merged**